setup.py

```diff
@@ -13,9 +13,10 @@
 
 install_requires = [
     'matplotlib>=3.2.0,<4',
-    'numpy>=1.18.0,<2',
-    'pandas>=1.1,<2',
-    'scipy>=1.4.1,<2',
+    "numpy>=1.18.0,<1.20.0;python_version<'3.7'",
```

Comment:
py3.6 still works with `numpy<2` right? Is there any reason we're capping it at `1.20.0`.

Reply:
Good point. I think I originally did this to test for sure it was working. I can expand the range again

Reply:
Sorry I got confused, python 3.6 doesn't work with anything after 1.20.0. That's the whole reason for this

Reply:
```
NumPy 1.20.0 Release Notes
This NumPy release is the largest so made to date, some 684 PRs
contributed by 184 people have been merged. See the list of highlights
below for more details. The Python versions supported for this release
are 3.7-3.9, support for Python 3.6 has been dropped. Highlights are

```